source/common/http/conn_manager_impl.cc

```diff
@@ -1045,7 +1051,29 @@ ConnectionManagerImpl::ActiveStream::commonEncodePrefix(ActiveStreamEncoderFilte
   if (!filter) {
     return encoder_filters_.begin();
   } else {
-    return std::next(filter->entry());
+    if (!always_start_next && (*(filter->entry()))->iterate_from_current_filter_) {
+      // The filter iteration has been stopped for all frame types, and now the iteration continues.
```

Comment:
Is it possible to assert we are stopped all here? Or has that state already been cleared? Same below?

Reply:
Yeah, it would have been cleared at the beginning of commonContinue().